admin/pre-reg/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.prereg, name='prereg'),
```

Comment:
Also, using a class-based view (the simple View class, not the generic CRUD ones) may help to group views for RESTful resources.